src/main/kotlin/com/google/androidstudiopoet/generators/packages/KotlinGenerator.kt

```diff
@@ -44,11 +44,38 @@ class KotlinGenerator constructor(fileWriter: FileWriter) : PackageGenerator(fil
         return blueprint.getMethodToCallFromOutside()
     }
 
+    override fun generateTestClass(blueprint: ClassBlueprint) {
+        val buff = StringBuilder()
+
+        buff.append("package ${blueprint.packageName};\n\n")
+
+        buff.append("import org.junit.Test\n\n")
+
+        buff.append("public class ${blueprint.className + "Test"} {\n")
```

Comment:
To avoid duplication between java and kotlin generators, please move `blueprint.className + "Test"` to the `ClassBlueprint`. Same for "MethodBlueprint" and test name.